library/ctr_drbg.c

```diff
@@ -51,6 +51,7 @@
 void mbedtls_ctr_drbg_init( mbedtls_ctr_drbg_context *ctx )
 {
     memset( ctx, 0, sizeof( mbedtls_ctr_drbg_context ) );
+    mbedtls_aes_init( &ctx->aes_ctx );
```

Comment:
With AES ALT implementations where `mbedtls_aes_init` and `mbedtls_aes_free` have to be paired, this fixes the sequence
```
mbedtls_ctr_drbg_init(&drbg);
mbedtls_ctr_drbg_free(&drbg);
```

However, it breaks the sequence
```
mbedtls_ctr_drbg_init(&drbg);
mbedtls_ctr_drbg_seed(&drbg);
…
```
since `mbedtls_aes_init(&drbg->aes_ctx)` is now called twice in a row.

`mbedtls_aes_init` has been called from the seed function since 8d128efd481396bfa9796bc1a78d386cac40bfd5 which split the initialization and the seeding of the DRBG into separate functions. I don't think there was a particular reason to defer the AES init to seed time, it was just a natural consequence of the order in which the code did things.

So please remove the call to `mbedtls_aes_init` in `mbedtls_ctr_drbg_seed`.